### PR TITLE
openvpn: only depend on tuntap on pre-yosemite os-x

### PR DESCRIPTION
**Pull request by lacombar**
Signed-off-by: Arnaud Lacombe <lacombar@gmail.com>

### CODE REVIEW

File: Library/Formula/openvpn.rb [hunk 1]
```diff
@@ -42,11 +42,19 @@ def install
     rm "#{share}/doc/openvpn/README.polarssl"
   end
 
-  def caveats; <<-EOS.undent
-    If you have installed the Tuntap dependency as a source package you will
-    need to follow the instructions found in `brew info tuntap`. If you have
-    installed the binary Tuntap package, no further action is necessary.
+  def caveats
+    s = ""
```
Comment (DomT4) [inline Library/Formula/openvpn.rb]: I think you need an `s` at the end of this block as well, for it to actually print. Maybe not. Your usage here is as echoed by Misty in the previous thread so I'll leave the review to Misty.

Comment (mikemcquaid) [inline Library/Formula/openvpn.rb]: Please push updates to this branch rather than creating a new PR. Thanks!

File: Library/Formula/openvpn.rb [hunk 2]
```diff
@@ -42,11 +42,19 @@ def install
...
+      s += <<-EOS.undent
+      If you have installed the Tuntap dependency as a source package you will
+      need to follow the instructions found in `brew info tuntap`. If you have
+      installed the binary Tuntap package, no further action is necessary.
+
+      EOS
+    end
+
+    s += <<-EOS.undent
     For OpenVPN to work as a server, you will need to create configuration file
```
Comment (mikemcquaid) [inline Library/Formula/openvpn.rb]: Ident this by two characters.

File: Library/Formula/openvpn.rb [hunk 3]
```diff
@@ -42,11 +42,19 @@ def install
...
-  def caveats; <<-EOS.undent
-    If you have installed the Tuntap dependency as a source package you will
-    need to follow the instructions found in `brew info tuntap`. If you have
-    installed the binary Tuntap package, no further action is necessary.
+  def caveats
+    s = ""
 
+    if MacOS.version < :yosemite
+      s += <<-EOS.undent
+      If you have installed the Tuntap dependency as a source package you will
```
Comment (mikemcquaid) [inline Library/Formula/openvpn.rb]: Ident this by another two characters

**Comment by mikemcquaid** [comment]
Tested this locally and looks like the `utun` stuff Just Works. Nice work here. Just waiting on a few more tweaks and we should be good to go.

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit a fresh PR as desired. Thanks for your submission to Homebrew!

**Comment by lacombar** [comment]
whitespace nits fixed in my branch, please re-open

**Comment by mikemcquaid** [comment]
@lacombar It won't let me; please create a new one, thanks!

**Comment by lacombar** [comment]
already did as #47161 :-)

FWIW, I really can't stand GitHub's "Pull requests" UI :-/ Really impractical...

**Pull request closed without merging**